### PR TITLE
[VEN-2686]: PSR deployment on zksync sepolia

### PR DESCRIPTION
**Pull request by GitGuru7**
## Description

Resolves #

### CODE REVIEW

File: deploy/001-psr.ts [hunk 1]
```diff
@@ -19,19 +19,29 @@ const func: DeployFunction = async ({
...
       execute: {
         methodName: "initialize",
         args: [acmAddress, loopsLimit],
       },
+      viaAdminContract: {
+        name: "DefaultProxyAdmin",
+        artifact: defaultProxyAdmin,
+      },
     },
+    // maxFeePerGas: "200000000"  // Needed for zksync
```
Comment (chechu) [inline deploy/001-psr.ts:44]: Could we set it conditionally considering the network?

File: hardhat.config.ts [hunk 2]
```diff
@@ -278,6 +279,13 @@ const config: HardhatUserConfig = {
     pages: "files",
     templates: "docgen-templates",
   },
+  dependencyCompiler: {
+    paths: [
+      "hardhat-deploy/solc_0.8/proxy/OptimizedTransparentUpgradeableProxy.sol",
+      "hardhat-deploy/solc_0.8/openzeppelin/proxy/transparent/ProxyAdmin.sol",
+      "hardhat-deploy/solc_0.8/openzeppelin/proxy/transparent/ProxyAdmin.sol:ProxyAdmin",
```
[the remark this answers is not in the document]
Comment (GitGuru7) [reply hardhat.config.ts:286]: Yes, we need this for backward compatibility

File: deploy/001-psr.ts (hunk 1 above)
Comment (GitGuru7) [reply deploy/001-psr.ts]: [ac7f495](https://github.com/VenusProtocol/protocol-reserve/pull/99/commits/ac7f49592b2d1795e464806534693510f940af0b)

File: hardhat.config.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (GitGuru7) [reply hardhat.config.ts]: [1f44f5c](https://github.com/VenusProtocol/protocol-reserve/pull/99/commits/1f44f5c495a7adf1d14ebd938a8aa83d348573f9)

**Review by chechu — APPROVED** [review]

**Pull request merged**